Hannah.gd

```diff
@@ -3,34 +3,47 @@ extends CharacterBody2D
 
 var speed = 300.0
 
+
+@onready var subviewport = $"../../../UI/PlayerInventoryUI/InvSprite/SubViewportContainer/SubViewport"
+
 #Get the inventory object - probably this should just be a child of the player? TODO refactor
 @onready var inventory = get_tree().get_nodes_in_group("Inventory")[0]
 var equipped
 var interacting = false
+var child_count_with_equipped 
```

Comment:
This isn't relevant to the end goal anymore

Reply:
On further thought - might still be needed but we can always add it back in if so

Reply:
https://github.com/StarkandcoEnterprises/Secret-game/pull/55/commits/b528671152748930312360d5526b6ed4517f5b04